package.json

```diff
@@ -34,7 +34,9 @@
 	"license": "ISC",
 	"private": true,
 	"dependencies": {
-		"@babylonjs/viewer": "^7.54.0",
+		"@babylonjs/viewer": "^8.2.0",
+		"@babylonjs/core": "^8.2.0",
+		"@babylonjs/loaders": "^8.2.0",
 		"@changesets/changelog-github": "^0.5.0",
```

Comment:
I think these should be in the `model3D` `package.json` rather than root.

Reply:
fixed.